inspirehep/modules/theme/static/scss/base/core.scss

```diff
@@ -18,13 +18,107 @@
  * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
  */
 
-/*
-Generic core styles that are included in all INSPIRE pages.
-*/
+// TODO: These will be moved to a new file
+article.header {
+  height: 28px;
+}
 
-/* ===========================================================
```

Comment:
The comment mentions what this file is for. So it should be kept and the styles related only to matching page moved to their own file.